ledger/common/convert/convert.go

```diff
@@ -18,6 +19,22 @@ const (
 // UnexpectedLedgerKeyFormat is returned when a ledger key is not in the expected format
 var UnexpectedLedgerKeyFormat = fmt.Errorf("unexpected ledger key format")
 
+// AddressToRegisterOwner converts 8-byte address to register owner.
+// If given address is ZeroAddress, it is a global register and owner is "".
+func AddressToRegisterOwner(address common.Address) string {
+	// global registers have an empty owner field
+	ownerString := ""
+
+	// all other registers have the account's address
+	if address != common.ZeroAddress {
+		ownerString = string(address.Bytes())
+	}
+
+	// all other registers have the account's address
+
+	return ownerString
```

Comment:
Maybe simplify this and avoid the assignment:

```suggestion
	// Global registers have address zero and an empty owner field
	if address == common.ZeroAddress {
		return ""
	}

	// All other registers have the account's address
	return string(address.Bytes())
```